framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -205,6 +248,32 @@ public void updateForMethodCall(
             MethodInvocationNode n, AnnotatedTypeFactory atypeFactory, V val) {
         ExecutableElement method = n.getTarget().getMethod();
 
+        List<String> sideEffectExpressions = null;
```

Comment:
Document that if the value is null, then there is no `@SideEffectsOnly` annotation.  You might want to include "only" in the variable name.